custom/enikshay/tasks.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+from collections import defaultdict
+from django.utils.dateparse import parse_datetime, parse_date
+
+from .case_utils import get_adherence_cases_between_dates_from_episode
+from corehq.apps.fixtures.models import FixtureDataItem
+
+
+DOMAIN = 'enikshay'
```

Comment:
This seems pretty brittle, and would be good to be able to use on our `enikshay-test` domain too. Maybe make this a feature flag thing?